persistence-cassandra-3.11/pom.xml

```diff
@@ -209,4 +233,56 @@
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>it-dse-6.8</id>
```

Comment:
Why are we declaring the profiles for the other persistence extension tests (we seem to do the same in other extensions pom)? Especially with no `executions`?

Reply:
I changed profile activation to properties and also made the `dse` profile activate if DSE integration tests are requested.

There's a bit of property params mess in `ci.yaml`, but at least I hope pom files are cleaner now.

Reply:
that ^ did not work, unfortunately. I guess I'll have to use some other approach :thinking:

Reply:
empty profiles have been removed in the latest push